.prettierrc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false,
+  "singleQuote": true,
+  "quoteProps": "as-needed",
+  "jsxSingleQuote": false,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "bracketSameLine": false,
+  "arrowParens": "always",
+  "proseWrap": "never",
+  "endOfLine": "lf",
+  "importOrder": ["<THIRD_PARTY_MODULES>", "^@xrengine/(.*)$", "^@mui/(.*)$", "^[./]"],
```

Comment:
need this to be `@etherealengine` instead of `@xrengine`

Reply:
on it

Reply:
done